omnibus/config/projects/inspec.rb

```diff
@@ -58,10 +58,8 @@
 
 package :rpm do
   signing_passphrase ENV['OMNIBUS_RPM_SIGNING_PASSPHRASE']
-  unless rhel? && platform_version.satisfies?('< 6')
-    compression_level 1
-    compression_type :xz
-  end
+  compression_level 1
+  compression_type :xz
```

Comment:
Should this be a separate commit? What's this about?

Reply:
We haven't built RHEL 5 packages for a while. This is a leftover from the olden days of our omnibus config.